backend/app/api/routes/scripts.py

```
@@ -165,180 +91,139 @@ async def get_node_content(
         
         if not has_access:
             raise HTTPException(status_code=403, detail="Access denied to this script")
-        
-        # Get current content
+
         content_lines = script["content_lines"]
         
-        # Validate line range
+        # Validate lines
         if start_line < 0 or end_line >= len(content_lines) or start_line > end_line:
             raise HTTPException(status_code=400, detail="Invalid line range")
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)